frontend/src/services/api.ts

```diff
@@ -62,53 +62,53 @@ export let getAdequacies = memoize(
   (params: PostAdequaciesRequest) => `${params.method}-${params.providers.join(',')}-${params.service_area_ids.join(',')}`
 )
 
-export async function getStaticRPs(selectedDataset: Dataset | null){
-    const res = await fetch(getStaticRPUrl(selectedDataset))
-    const body = await res.json()
-    return body as PostRepresentativePointsResponse
+export async function getStaticRPs(selectedDataset: Dataset | null) {
```

Comment:
Can we agree on the whitespace settings for use in VS Code? And possibly have a single PR just to change the whitespace in all the files?